README.md

```diff
@@ -1,20 +1,41 @@
-majkrzak/create-release
+Andrew-Kulpa/create-release
 =======================
 
 Github Action for handling release creation
 
 
+Build Process
+-------------
+```
+  $ npm install
+  $ npm run build
+  
+  $ git checkout -b releases/v{version_number}
+  $ git commit -a -m "prod dependencies"
+
+  $ npm prune --production
+  $ git add node_modules
+  $ git commit -a -m "prod dependencies"
+  $ git push origin releases/v{version_number}
+```
+
+
 Example
 -------
 
 ```yaml
-    - uses: majkrzak/create-release@master
+    - name: Release Executables
+      uses: andrew-kulpa/create-release@v1
       with:
-        token: ${{github.token}}
-        name: My lovely release
-        code: latest
-        prerelease: true
-        assets: >
-          source.txt:target.txt:text/plain
-          another:one:application/json
+        name: Your release name
```

Comment:
Might be better to provide linux and windows both as example. Mostly stuff are begin built on linux.